app/perfunwind.cpp

```diff
@@ -456,6 +456,10 @@ void PerfUnwind::resolveCallchain()
     // in the normal callchain. The branch stack contains the non-kernel IPs then.
     const bool hasBranchStack = !m_currentUnwind.sample->branchStack().isEmpty();
 
+    if (m_currentUnwind.sample->callchain().size() > maxUnwindStack()) {
+        m_currentUnwind.isIncompleteCallchain = true;
```

Comment:
ideally this also gets set when the stack sample is too small and unwinding fails for a deep call stack